clusterloader2/pkg/measurement/common/api_availability_measurement.go

```diff
@@ -252,7 +259,10 @@ func (a *apiAvailabilityMeasurement) gather() ([]measurement.Summary, error) {
 	}
 	summary := measurement.CreateSummary(apiAvailabilityMeasurementName, "json", content)
 	a.summaries = append(a.summaries, summary)
-	return a.summaries, nil
+	if sli := output.ClusterSummary.AvailabilityPercentage; sli < a.threshold {
```

Comment:
are you comparing percentage with a fraction?

Reply:
Also: having percentage in the summary and fraction in an error and variable is inconsistent

Reply:
>are you comparing percentage with a fraction?
>Also: having percentage in the summary and fraction in an error and variable is inconsistent

Nope, this is a comparison of percentages. `threshold` parameter is also provided as a percentage in the config.

Reply:
right
`CL2_API_AVAILABILITY_THRESHOLD 0.0` suggested otherwise
Maybe `CL2_API_AVAILABILITY_PERCENTAGE_THRESHOLD` then, to make it more explicit?

Reply:
Sounds good! Done.